src/deluge/model/note/note_row.cpp

```diff
@@ -2965,7 +2965,7 @@ int32_t NoteRow::readFromFile(int32_t* minY, InstrumentClip* parentClip, Song* s
 				if (velocity == 0 || velocity > 127) {
 					velocity = 64;
 				}
-				if ((probability & 127) > (kNumProbabilityValues + 35) || (probability & 127) == 0
+				if ((probability & 127) > (kNumProbabilityValues + kNumIterationValues)
```

Comment:
is dropping the (probabilty & 127) == 0 condition intentional?

Reply:
Yes, for the same reason as above - 0 is the value fro fill.